doc/cephadm/services/index.rst

```diff
@@ -2,10 +2,26 @@
 Service Management
 ==================
 
+A service is a group of daemons configured together. See these chapters
+for details of how do deploy individual services:
```

Comment:
```suggestion
A service is a group of daemons configured together. See these chapters
for details on individual services:
```
I feel like some of these sections cover a lot more than just deploying the services and this sounds a bit nicer.